TsRandomizer/Screens/SaveSelectScreen.cs

```diff
@@ -318,9 +318,51 @@ static void WriteItemListSection(StreamWriter file, IEnumerable<ItemLocation> it
 		static void WriteItemList(StreamWriter file, IEnumerable<ItemLocation> itemLocations, int depth)
 		{
 			var prefix = new string('\t', depth);
+			string itemName = "";
 
 			foreach (var itemLocation in itemLocations)
-				file.WriteLine(prefix + itemLocation);
+			{
+				if (itemLocation.ItemInfo is ProgressiveItemInfo)
+				{
+					itemName = "Progressive item:";
+					foreach (var item in itemLocation.ItemInfo.AsDynamic().Items)
+						itemName += " | " + GetItemName(item.Identifier);
```

Comment:
So a few comments about this, first off all you should not use .AsDynamic() to access members of classes that are under our own controll, you can just make Items public or internal
Secondly i think i prefered the > over the | as seperator as it points our the progression path

but more importantly this functionality was originally done in the .tostring() of the iteminfo itzelf, therefor i recommend moving the GetItemName() to the the ItemIdentifier class, and just use the tostring() methods of the iteminfo and PogRessiveitemInfo

PogRessiveitemInfo was not a typo btw, its a twitch thing because these progressive items are so POG! but i am ok with the rename if you also rename the PogRessiveitemInfo.cs aswel